### ISSUE TITLE
"Asus Keyboard" Device not always appear in /input/devices, on Rog Ally.

### ISSUE THREAD

**Issue by fkysly** [Maclay74/steam-patch#12]
ok, I found the problem. Because the "Asus Keyboard" Hid Device not always appear in /input/devices, so steam-patch not recognize it when "Asus Keyboard" not appear.

I: Bus=0003 Vendor=0b05 product=1abe Version=0110
N: Name="Asus Keyboard"
...

On my Rog Ally, suspend will disappear this, so when awake up, press the Main Button or QAM Button not responsible everytime.

I don't know why cause this.

**Comment by fkysly** [comment]
After uninstall the AC SE App on Windows, Asus Keyboard return back on ChimeraOS, But there are three "Asus Keyboard" Device on evtest list, only the last will response for the main & QAM button, I need to modify the steam-patch relative part...

**Comment by fkysly** [comment]
``` rust
if device.supported_keys().map_or(false, |keys| keys.contains(Key::KEY_ENTER)) {
  println!("are you prepared to ENTER the world of evdev?");
}
```

should we use the method to recognize the asus keyboard rather than the id?

**Comment by Maclay74** [comment]
Why do you think it's more reliable? 
As far as I know, after a full power cycle you'll have this keyboard back. That's something ChimeraOS team is focused on.

**Comment by fkysly** [comment]
> Why do you think it's more reliable? As far as I know, after a full power cycle you'll have this keyboard back. That's something ChimeraOS team is focused on.

yes, uninstall windows SE, seems like some keyboard driver also uninstall from bios. So it's more reliable than before, it's always has the asus keyboard device, includes resume from suspend.

The only different is that this method will have three asus keyboard device, so must update steam-patch to regonize the right one. modify use the key equal method should fix this problem.
